core/scheduler.py

```diff
@@ -32,51 +32,66 @@ def getInstance():
             DMLScheduler()
         return DMLScheduler.__instance
 
-    def __init__(self):
+    #Default number of runners that are running at a time is 4
```

Comment:
These kind of comments are better inside docstrings rather than outside methods. But I actually think we should remove it since the code is pretty much self-documenting at the function definition.

Reply:
Removed.